src/styles/text/text.js

```diff
@@ -92,15 +92,32 @@ Object.assign(TextStyle, {
             return;
         }
 
-        q.feature = feature;
-        q.context = context;
-        q.layout.vertex = false; // vertex placement option not applicable to standalone labels
+        // text can be an array if a `left` or `right` orientation key is defined for the text source
+        // in which case, push both text sources to the queue
+        if (q instanceof Array){
```

Comment:
Probably not. We do some weird things with iframes in tools like Differ, but I don't think that's relevant for the cases we're discussing here (and not part of the core library anyway).